### PR TITLE
Remove agent state

### PR DESCRIPTION
**Pull request by uohzxela**

### CODE REVIEW

File: lib/ex_format/comments.ex [hunk 1]
```diff
@@ -1,50 +1,42 @@
...
+    inline_comments
+  end
+
+  defp update_inline_comments(inline_comments, k, v) do
+    if Map.has_key?(inline_comments, k) do
+      val = Map.get(inline_comments, k)
+      Map.put(inline_comments, k, val ++ [v])
+    else
+      Map.put(inline_comments, k, [v])
+    end
```
Comment (whatyouhide) [inline lib/ex_format/comments.ex]: The whole `if` can be replaced with

```elixir
Map.update(inline_comments, k, [v], &(&1 ++ [v]))
```

File: lib/ex_format/lines.ex [hunk 2]
```diff
@@ -1,25 +1,13 @@
...
-  def initialize_lines_store(code_string) do
-   start_link(code_string)
+  def initialize_lines_map(code_string) do
+    code_string
+    |> String.split("\n")
+    |> Enum.with_index()
+    |> Enum.map(fn {line, i} ->
+      {i + 1, String.trim(line)}
+    end)
+    |> Map.new
```
Comment (whatyouhide) [inline lib/ex_format/lines.ex]: Can be replaced with

```elixir
lines = String.split(code_string, "\n")
for {line, index} <- Enum.with_index(lines, _offset = 1),
    into: %{},
    do: {index, String.trim(line)}
```

**Review by whatyouhide — CHANGES_REQUESTED** [review]

File: lib/ex_format/comments.ex (hunk 1 above)
Comment (uohzxela) [reply lib/ex_format/comments.ex]: Elegant! That worked perfectly.

File: lib/ex_format/lines.ex (hunk 2 above)
Comment (uohzxela) [reply lib/ex_format/lines.ex]: Thank you, this looks way better.

**Comment by uohzxela** [comment]
@whatyouhide updated and ready to merge :)

**Review by whatyouhide — APPROVED** [review]

**Pull request merged**